src/main/java/edu/kis/powp/jobs2d/drivers/adapter/TransformerDriver.java

```diff
@@ -0,0 +1,35 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.transformers.ComplexTransformerCommand;
+import edu.kis.powp.jobs2d.command.transformers.TransformedCoords;
+import edu.kis.powp.jobs2d.command.transformers.TransformerCommand;
+
+public class TransformerDriver implements Job2dDriver {
+    private Job2dDriver job2dDriver;
+    private ComplexTransformerCommand transformerCommands;
```

Comment:
It seems that using an interface TransformerCommand type instead of ComplexTransformerCommand is possible here (low coupling)